sentinelhub/constants.py

```diff
@@ -54,6 +54,22 @@ class ServiceType(Enum):
     PROCESSING_API = "processing"
 
 
+class ResamplingType(Enum):
```

Comment:
While checking if service is case-sensitive about these parameters I discovered this:

```
"possibleValues":["BILINEAR","BICUBIC","LANCZOS","BOX","NEAREST"]
```

It turns out the 2 missing options work but they are not written in SH documentation. I'm not completely sure but I think it's better that we include them in the enum.

Reply:
Please keep the possible values aligned with docs. Anything else is not officially supported, so we (sh-py) should not be exposing it.

Reply:
Ok, then let's just keep the official ones.